### PR TITLE
Update ML.NET CLI Install Guide

### LINKED ISSUE #24838: Update How to install the ML.NET Command-Line Interface (CLI) tool
- Add note about bash
- Review outdated content and update as necessary

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4200d7af-ea16-7b93-5263-6a1ce41ecb3b
* Version Independent ID: e1697ee4-9d3a-e1cc-b678-cab88f1b5f86
* Content: [How to install the ML.NET Command-Line Interface (CLI) tool - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-ml-net-cli)
* Content Source: [docs/machine-learning/how-to-guides/install-ml-net-cli.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/how-to-guides/install-ml-net-cli.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by jwood803**
- Add note about zsh

Fixes #24838

[Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-ml-net-cli?branch=pr-en-us-25054)

AB#1864084

### CODE REVIEW

**Comment by luisquintanilla** [comment]
Blocked by dotnet/machinelearning-modelbuilder#1585

**Comment by jwood803** [comment]
@luisquintanilla This should now be good for review.

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 1]
```diff
@@ -51,6 +51,14 @@ mlnet
 
 You should see the help for available commands for the mlnet tool such as the 'classification' command.
 
+If you are running Linux or macOS, note that if you're using a console other than Bash (for example, zsh, which is the new default for macOS), then you'll need to give `mlnet` executable permissions and include `mlnet` to the system path. Instructions on how to do this should appear in the terminal when you install `mlnet` (or any global tool).
+
+Alternatively, you can try using the following command to run the mlnet tool:
+
+```console
+~/.dotnet/tools/mlnet
+```
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:60]:
```suggestion
> [!IMPORTANT]
> If you are running Linux or macOS, note that if you're using a console other than Bash (for example, zsh, which is the new default for macOS), then you'll need to give `mlnet` executable permissions and include `mlnet` to the system path. Instructions on how to do this should appear in the terminal when you install `mlnet` (or any global tool).
>
> Alternatively, you can try using the following command to run the mlnet tool:
> 
> ```console
> ~/.dotnet/tools/mlnet
> ```
```

**Review by luisquintanilla — COMMENTED** [review]
Looks great. Added IMPORTANT to highlight zsh instructions.

File: docs/machine-learning/how-to-guides/install-ml-net-cli.md [hunk 2]
```diff
@@ -51,6 +51,15 @@ mlnet
 
 You should see the help for available commands for the mlnet tool such as the 'classification' command.
 
+> [!IMPORTANT]
+> If you are running Linux or macOS, note that if you're using a console other than Bash (for example, zsh, which is the new default for macOS), then you'll need to give `mlnet` executable permissions and include `mlnet` to the system path. Instructions on how to do this should appear in the terminal when you install `mlnet` (or any global tool).
+>
+> Alternatively, you can try using the following command to run the mlnet tool:
+>
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/install-ml-net-cli.md:58]:
```suggestion
>
```

Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/install-ml-net-cli.md:58]: Fixing MD lint errors

**Review by luisquintanilla — COMMENTED** [review]
@jwood803 added a suggestion to fix MD lint errors

**Review by luisquintanilla — APPROVED** [review]
Looks good. Thanks @jwood803

**Pull request merged**